src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```
@@ -37,6 +37,12 @@
 
   <PropertyGroup>
     <IncludeMultiTargetRoslynComponentTargets>false</IncludeMultiTargetRoslynComponentTargets>
-    <RoslynComponentsAreOffByDefault>true</RoslynComponentsAreOffByDefault>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="buildTransitive\$(MSBuildProjectName).*"
```

```suggestion
    <Content Include="buildTransitive\$(PackageId).targets"
```
nit: I tend to avoid a wildcard unless it's needed since it hurts evaluation perf.  nit: `$(PackageId)` should be used rather than `$(MSBuildProjectName)` since they can differ, but the PackageID is the one that needs to match.

---

Maybe we could update the docs (and the other places that do this as well) to be consistent across the repo?

https://github.com/dotnet/runtime/blob/4f4dd93eca92aca3c6bd49a5b9c2eca908112c32/docs/coding-guidelines/libraries-packaging.md?plain=1#L130-L137

I believe the `*` is there to support both `.props` and `.targets`. But @ViktorHofer would know for sure.